policyengine_us/variables/gov/ed/pell_grant/pell_grant.py

```diff
@@ -3,8 +3,19 @@
 
 class pell_grant(Variable):
     value_type = float
-    entity = SPMUnit
+    entity = Person
     label = "Pell Grant amount"
     documentation = "SPM unit's Pell Grant educational subsidy"
     definition_period = YEAR
     unit = USD
+
+    def formula(person, period, parameters):
+        coa = person("pell_grant_cost_of_attendance", period)
+        months_in_school = person("pell_grant_months_in_school", period)
+        efc = person("pell_grant_efc", period)
+        p = parameters(period).gov.ed.pell_grant
+        minimum = p.min_grant
+        maximum = p.max_grant
+        amount = min_(coa - efc, maximum)
```

Comment:
We generally avoid reusing Python variables. You can use the params directly.
```suggestion
        unbounded = coa - efc
        capped = min_(unbounded, p.amount.max)
```